bin/get-pgmonitor.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -e
+
+# Copyright 2017 - 2020 Crunchy Data Solutions, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "Getting pgMonitor..."
+PGMONITOR_COMMIT='v4.4-RC2'
+
+# pgMonitor Setup
+if [[ -d ${PGOROOT?}/tools/pgmonitor ]]
+then
+    rm -rf ${PGOROOT?}/tools/pgmonitor
+fi
+
+git clone https://github.com/CrunchyData/pgmonitor.git ${PGOROOT?}/tools/pgmonitor
+cd ${PGOROOT?}/tools/pgmonitor
+git checkout ${PGMONITOR_COMMIT?}
+
+# create pgMonitor documentation data directory, if it doesn't exist
+if [[ ! -d ${PGOROOT?}/docs/data/pgmonitor/general ]]
+then
+    mkdir -p ${PGOROOT?}/docs/data/pgmonitor/general
+fi
+
+# pgMonitor Script Documentation Setup
+rm -rf ${PGOROOT?}/docs/data/pgmonitor/general/*
+
+# create pgMonitor documentation data directory, if it doesn't exist
+if [[ ! -d ${PGOROOT?}/docs/data/pgmonitor/pgnodemx ]]
+then
+    mkdir -p ${PGOROOT?}/docs/data/pgmonitor/pgnodemx
+fi
+
+# pgMonitor Script Documentation Setup
+rm -rf ${PGOROOT?}/docs/data/pgmonitor/pgnodemx/*
+
+# link pgMonitor metrics files to Hugo data directory
+ln -s $PGOROOT/tools/pgmonitor/exporter/postgres/queries_common.yml $PGOROOT/docs/data/pgmonitor/general/queries_common.yml
+ln -s $PGOROOT/tools/pgmonitor/exporter/postgres/queries_per_db.yml $PGOROOT/docs/data/pgmonitor/general/queries_per_db.yml
+ln -s $PGOROOT/tools/pgmonitor/exporter/postgres/queries_backrest.yml $PGOROOT/docs/data/pgmonitor/general/queries_backrest.yml
+ln -s $PGOROOT/tools/pgmonitor/exporter/postgres/queries_pg12.yml $PGOROOT/docs/data/pgmonitor/general/queries_pg12.yml
+ln -s $PGOROOT/tools/pgmonitor/exporter/postgres/queries_pg11.yml $PGOROOT/docs/data/pgmonitor/general/queries_pg11.yml
+ln -s $PGOROOT/tools/pgmonitor/exporter/postgres/queries_pg10.yml $PGOROOT/docs/data/pgmonitor/general/queries_pg10.yml
+ln -s $PGOROOT/tools/pgmonitor/exporter/postgres/queries_pg96.yml $PGOROOT/docs/data/pgmonitor/general/queries_pg96.yml
+ln -s $PGOROOT/tools/pgmonitor/exporter/postgres/queries_pg95.yml $PGOROOT/docs/data/pgmonitor/general/queries_pg95.yml
+ln -s $PGOROOT/tools/pgmonitor/exporter/postgres/queries_nodemx.yml $PGOROOT/docs/data/pgmonitor/pgnodemx/queries_nodemx.yml
```

Comment:
Can these be relative symlinks? If so, we can commit them to Git and eliminate most of this script.

```
docs/data/pgmonitor/general/queries_common.yml -> ../../../../tools/pgmonitor/exporter/postgres/queries_common.yml
```

Reply:
Yes, I like that idea! Updated, thanks.